src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -206,43 +166,25 @@ public void stopFixedArm() {
         setMotor(ClimbConstants.STOP_SPEED, climbFixedMotor);
```

Comment:
`setMotor()`'s `value` parameter is position in encoder ticks, not percent output.